shaders/palette.frag

```diff
@@ -85,10 +88,28 @@ void main() {
         texel.r = clamp(texel.r + offset, 0, limit);
     }
 
-    bool NO_REMAP = USE_HAR_QUIRKS && index > 0x30;
+    bool NO_REMAP = SPRITE_HAR_QUIRKS && index > 0x30;
+
+    // chronos's stasis & shadow's shadow transparency
+    if(SPRITE_LOOPBACK) {
+        index = int(texel.r * 255.0);
+        if(index < 0x60) {
+            // grab color index we're drawing ontop of
+            int behind = int(texture(loopback, gl_FragCoord.xy / NATIVE_SIZE).r * 255.0);
+            // lookup behind in remap 5 to get brightness
+            vec4 remap = texture(remaps, vec2(float(behind) / 255.0, 4.0 / 18.0));
+            behind = 1 + clamp(int(remap.r * 255.0) - 0xA8, 0, 7) * 2;
+
+            // this indexed blending is not exactly correct, but i am
+            // done fiddling with it for now-- if someone finds the
+            // code for this in MASTER.DAT, then we can correct it.
+            index = (index & 0xF0) + ((index & 0x0F) * 3 + behind * 2) / 5;
+        }
+        texel.r = float(index) / 255.0;
+    }
```

Comment:
This seems like something that should be done in rgba shader. Then you would have the actual finished background image instead of the possibly half-baked one here ? Also, glCopyTexSubImage2D has a bad tendency to stall.

Reply:
rgba shader would need access to both the HAR's `index` and the `index` beneath it.
could be done, but we'd need to rethink our texture channels.

Reply:
To clear up some texture channels:
- Pack remap into one byte `color.g = float(remap_index + remap_count * 19) / 255.0;`, which works as long as remap_count isn't greater than 13.
- Hot-bunk a channel between SPRITE_INDEX_ADD  ('bg' tag is only used in credits) and DARK_TINT (only used in arena).

Brings us down to three channels, in case we need the fourth for something in the future.

Reply:
I wonder if just throwing in another texture as render target would work. Theoretically possible, but never tested it.

Reply:
Shuffled the data around.
I didn't implement the SPRITE_INDEX_ADD/SPRITE_DARK_TINT hot-bunking idea, but if we need another channel that will still save us one.

Reply:
Stopped deferring SPRITE_INDEX_ADD via the alpha channel-- it's now written directly to the red channel.
Alpha channel's free in case we ever need it

Reply:
Yep. The additive blending thing was the reason why the whole two-stage pipeline was made. We can't add in the palette stage, because we need to do the other operations first, and because it's not actually additive blending; it's index addition with conditions :P

Reply:
Freed up the SPRITE_INDEX_ADD color channel when not in SCENE_CREDITS, so we're currently up one channel.
woop!